python/mlx/nn/layers/linear.py

```diff
@@ -7,12 +7,20 @@
 
 
 class Linear(Module):
-    """Applies an affine transformation to the input.
+    """Applies a affine transformation to the input.
```

Comment:
Nit: `a affine` -> `an affine` and also I would rather do a raw docstring literal rather than a double backslash.